src/haz3lcore/dynamics/Builtins.re

```diff
@@ -33,6 +33,40 @@ module Pervasives = {
   module Impls = {
     open EvaluatorMonad;
     open EvaluatorResult;
+
+    /* is_finite implementation. */
+    let is_finite = (name, r1) =>
+      switch (r1) {
+      | BoxedValue(FloatLit(f)) =>
+        let b = Float.is_finite(f);
+        BoxedValue(BoolLit(b)) |> return;
+      | BoxedValue(d1) =>
+        raise(EvaluatorError.Exception(InvalidBoxedBoolLit(d1)))
```

Comment:
shouldn't raise an InvalidBoxedBoolLit error

Reply:
What kind of error would it raise? Would it be an InvalidOperation error?